Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml

```diff
@@ -0,0 +1,1268 @@
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              simple: inputs.Hostname
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 215
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "4":
+    id: "4"
+    taskid: ea90c16b-6985-4f28-816f-78608df3fe51
+    type: title
+    task:
+      id: ea90c16b-6985-4f28-816f-78608df3fe51
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1470
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "8":
+    id: "8"
+    taskid: 9fa921fa-d196-40ba-8419-ed0c4f838ab8
+    type: condition
+    task:
+      id: 9fa921fa-d196-40ba-8419-ed0c4f838ab8
+      version: -1
+      name: Is Carbon Black Enterprise Response enabled?
+      description: Checks if there is an active instance of the Carbon Black Enterprise Response integration enabled.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "9"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: containsGeneral
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: VMware Carbon Black EDR v2
+                    ignorecase: true
+                accessor: state
+            iscontext: true
+          right:
+            value:
+              simple: active
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 950,
+          "y": 710
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 79f17c3a-2456-47d9-8e68-cc3d84ef01bd
+    type: regular
+    task:
+      id: 79f17c3a-2456-47d9-8e68-cc3d84ef01bd
+      version: -1
+      name: Get host information from Carbon Black Enterprise Response
+      description: List the CarbonBlack sensors
+      script: VMware Carbon Black EDR v2|||endpoint
```

Comment:
Same here, why we are using endpoint again?

Reply:
`endpoint` is a generic command and it must be `endpoint (VMware Carbon Black EDR v2)` on this branch.